tests/providers/google/cloud/hooks/test_datafusion.py

```diff
@@ -197,13 +196,6 @@ def test_start_pipeline(self, mock_wait_for_pipeline_state, mock_request, hook):
         mock_request.assert_called_once_with(
             url=f"{INSTANCE_URL}/v3/namespaces/default/start", method="POST", body=body
         )
-        mock_wait_for_pipeline_state.assert_called_once_with(
-            instance_url=INSTANCE_URL,
-            namespace="default",
-            pipeline_name=PIPELINE_NAME,
-            pipeline_id=run_id,
-            success_states=SUCCESS_STATES + [PipelineStates.RUNNING],
-        )
```

Comment:
Since we removed this, should a case be added on the operator tests to make sure this is called?

Reply:
Test extended @uranusjr